third_party/nvfuser/csrc/mutator.cpp

```diff
@@ -121,6 +121,10 @@ void OptOutMutator::mutate(kir::TensorIndex*) {
   TORCH_INTERNAL_ASSERT(false, "Not implemented yet.");
 }
 
+void OptOutMutator::mutate(AggregateVal*) {
+  TORCH_INTERNAL_ASSERT(false, "Not implemented yet.");
+}
```

Comment:
I don't think this is needed anymore.

Reply:
I agree

Reply:
actually I get a compile error if I remove it

Reply:
solved. I removed mutate from here and dispatch.h